### ISSUE TITLE
Unable to detect the objects in high resolution images using tensorflow object detection API 

### ISSUE THREAD

**Issue by snaveen2592** [tensorflow/models#8465]
Hi,
I am unable to find all the objects using tensorflow object detection API
Please find the below  deteils :
Tensorflow version : 1.9.0
training steps : 50000, 100000, 1.50 lakhs steps
learning rate :0.00002
Batch Size : 1
Pretrained Model : faster RCNN inception, faster RCNN resnet 101 coco, rFCN resnet 101 coco
Image size : 3507 X 2958 resolution
DPI : 200
GPU memory : NVIDIA QUADRO P1000(4 GB)
Please suggest the pretrained models for better accuracy in high resolution images

**Comment by pkulzc** [comment]
If you take a look at our code, you will see that our model always resize the input to a fixed size(e.g. 320x320, 640x640..) so your image size isn't the the size the model will see. This is a very common thing for all object detection models. 

Our object detection models are fully convolutional so there is still a way to do this:
- modify the resizer field in config file to your ideal sizes (e.g. 1024x1024). 
- use export_inference_graph to reexport the model
- run inference with your newly exported model

Note that using input size greater than 1000 could easily used up all your memory!
